Team viewer collector (#25796)
demisto/content: Packs/TeamViewer/Integrations/TeamViewerEventCollector/TeamViewerEventCollector_description.md

## TeamViewer
Use this integration to collect events automatically from TeamViewer.
You can also use the ***teamviewer-get-events*** command to manually collect events.

### Script Token
To use the TeamViewer collector, you would need to create a script token.

1. Log in to the Management Console: https://login.teamviewer.com/

> 2. Click on your user in the upper right corner and select Edit profile.

Reviewer: ```suggestion
2. Click on your user in the upper right corner and select **Edit profile**.
```

> 3. Select Apps.

Reviewer: ```suggestion
3. Select **Apps**.
```

> 4. Select Create script token.

Reviewer: ```suggestion
4. Select **Create script token**.
```

5. Tick all the permissions you need for your token.

> 6. Select Create.

Reviewer: ```suggestion
6. Select **Create**.
```

> ## Team Viewer

Reviewer: ```suggestion
## TeamViewer
```

> Use this integration to collect events automatically from Team Viewer.

Reviewer: ```suggestion
Use this integration to collect events automatically from TeamViewer.
```

You can also use the ***teamviewer-get-events*** command to manually collect events.

### Script Token

> [For instructions on how to generate your script token, click here](https://community.teamviewer.com/English/kb/articles/109647-use-the-teamviewer-api).

Reviewer 2: Please add here the steps to create the script token and then add the link (for more information....)
